.github/workflows/build.yml

```diff
@@ -133,7 +133,7 @@ jobs:
         DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
       run: |
         echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
-        docker push antrea/antrea-windows:latest
+        make build-and-push-windows
```

Comment:
When the main branch is updated, this job will run both `make build-windows` and `make build-windows-push`. Maybe we should add this condition to the previous step: `if: ${{ github.repository != 'antrea-io/antrea' || github.event_name != 'push' || github.ref != 'refs/heads/main' }}`

Reply:
Do you mean you want me to add the opposite condition of push to trigger command `make build-windows`? The target logic is like this,
```
if: ${{ github.repository == 'antrea-io/antrea' && github.event_name == 'push' && github.ref == 'refs/heads/main' }}
     make build-and-push-windows
else:
     make build-windows
```

Reply:
This reminds me a candidate issue, what is the antrea-windows image tag when pushing to antrea  repository? `latest` or a hash value?  Now I use 'latest' when trying to save the image locally, which is consumed by CI; and use `${DOCKER_IMG_VERSION}` when pushing to remote registry. It seems incorrect. @antoninbas

Reply:
> Do you mean you want me to add the opposite condition of push to trigger command

Yes, you can refer to the Linux build in the same file

> This reminds me a candidate issue, what is the antrea-windows image tag when pushing to antrea repository?

The tag should be `latest` when pushing to the registry (for the `main` branch build), same as for Linux